vmdb/spec/lib/miq_automation_engine/miq_ae_state_machine_new_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+require 'timecop'
+
+describe "MiqAeStateMachine" do
+  before do
+    class DummyWorkspace
```

Comment:
Need to leverage `Class.new` vs defining a class here.

```ruby
let(:test_class) do
  Class.new()
  ...
end
```

@gmcculloug I can take this commit and create a new PR with the changes (keeping Madhu's name) or we can merge and I can write a followup PR that will remove this class.